### PR TITLE
fix the readme_en

### PR DESCRIPTION
**Pull request by lilith-zy**
fix the readme_en

### CODE REVIEW

File: README_en.md [hunk 1]
```diff
@@ -4,9 +4,9 @@
 
 ## Introduction
 
-PaddleClas is a toolset for image classification tasks prepared for the industry and academia. It helps users train better computer vision models and apply them in real scenarios.
+PaddleClas is a toolset for image recognition tasks prepared for the industry and academia helping users train better computer vision models and apply them in real scenarios.
```
Comment (weisy11) [inline README_en.md:7]: PaddleClas is a image recognition toolset for industry and academia, helping...

File: README_en.md [hunk 2]
```diff
@@ -21,467 +21,97 @@ PaddleClas is a toolset for image classification tasks prepared for the industry
 
 - [more](./docs/en/update_history_en.md)
 
-
 ## Features
 
-- Rich model zoo. Based on the ImageNet-1k classification dataset, PaddleClas provides 29 series of classification network structures and training configurations, 134 models' pretrained weights and their evaluation metrics.
-
-- SSLD Knowledge Distillation. Based on this SSLD distillation strategy, the top-1 acc of the distilled model is generally increased by more than 3%.
+- A practical image recognition system: Integrate modules for detection, feature learning and retrieval, widely applicable to all types of image recognition tasks.
```
Comment (weisy11) [inline README_en.md:26]: A practical image recognition system consist of detection, feature learning and retrieval modules, ...

File: README_en.md [hunk 3]
```diff
@@ -21,467 +21,97 @@ PaddleClas is a toolset for image classification tasks prepared for the industry
...
+Four sample solutions are provided, including product recognition, vehicle recognition, logo recognition and animation character recognition.
 
-- Data augmentation: PaddleClas provides detailed introduction of 8 data augmentation algorithms such as AutoAugment, Cutout, Cutmix, code reproduction and effect evaluation in a unified experimental environment.
+- Rich library of pre-trained models: Provide a total of 134 ImageNet pre-trained models in 29 series, with 6 selected series of models supporting fast structural modification.
```
Comment (weisy11) [inline README_en.md:30]: 29->35 134->164

Comment (weisy11) [reply README_en.md:30]: 这个with 有点奇怪，因为6个是29个系列中的，不是额外的，among which 会好一点吗？

File: README_en.md [hunk 4]
```diff
@@ -21,467 +21,97 @@ PaddleClas is a toolset for image classification tasks prepared for the industry
...
 
-- Pretrained model with 100,000 categories: Based on `ResNet50_vd` model, Baidu open sourced the `ResNet50_vd` pretrained model trained on a 100,000-category dataset. In some practical scenarios, the accuracy based on the pretrained weights can be increased by up to 30%.
+- Comprehensive and easy-to-use feature learning components: A large number of metric learning methods are integrated and can be combined and switched at will through configuration files.
```
Comment (weisy11) [inline README_en.md:31]: a large number of -> 12

File: README_en.md [hunk 5]
```diff
@@ -21,467 +21,97 @@ PaddleClas is a toolset for image classification tasks prepared for the industry
...
 
-- A variety of training modes, including multi-machine training, mixed precision training, etc.
+- SSLD knowledge distillation: the recognition accuracy of distilled models based on this solution is generally improved by more than 3%.
```
Comment (weisy11) [inline README_en.md:34]: 这个中文的部分整个都变了

File: README_en.md [hunk 6]
```diff
@@ -21,467 +21,97 @@ PaddleClas is a toolset for image classification tasks prepared for the industry
...
 
-- A variety of inference and deployment solutions, including TensorRT inference, Paddle-Lite inference, model service deployment, model quantification, Paddle Hub, etc.
+- Data augmentation: Support for 8 data augmentation algorithms such as AutoAugment, Cutout, Cutmix, etc.  with detailed introduction, code replication and evaluation of effectiveness in a unified experimental environment.
```
Comment (weisy11) [inline README_en.md:36]: support for -> providing ？这里的支持是我们提供了这些算法的意思，不是对外的支持

File: README_en.md [hunk 7]
```diff
@@ -21,467 +21,97 @@ PaddleClas is a toolset for image classification tasks prepared for the industry
...
-Accuracy and inference time metrics of AlexNet, SqueezeNet series, VGG series and DarkNet53 models are shown as follows. More detailed information can be refered to [Others](./docs/en/models/Others_en.md).
-
+- [Quick Installatiopn](./docs/zh_CN/tutorials/install.md)
+- [Quick Start of Recognition](./docs/zh_CN/tutorials/quick_start_recognition.md)
+- Algorithms Introduction（Updating）
+    - [Backbone Network and Pre-trained Model Library](./docs/zh_CN/models/models_intro.md)
+    - [Mainbody Detection](./docs/zh_CN/application/object_detection.md)
+    - Image Classification
+        - [ImageNet Classification](./docs/zh_CN/tutorials/quick_start_professional.md)
+    - 特征学习
```
Comment (weisy11) [inline README_en.md:64]: 这咋还有个中文

File: README_en.md [hunk 8]
```diff
@@ -21,467 +21,97 @@ PaddleClas is a toolset for image classification tasks prepared for the industry
...
+        - [Product Recognition](./docs/zh_CN/application/product_recognition.md)
+        - [Vehicle Recognition](./docs/zh_CN/application/vehicle_reid.md)
+        - [Logo Recognition](./docs/zh_CN/application/logo_recognition.md)
+        - [Animation Character Recognition](./docs/zh_CN/application/cartoon_character_recognition.md)
+    - [Vector Retrieval](./deploy/vector_search/README.md)
+- Models Training/Evaluation
+    - [Image Classification](./docs/zh_CN/tutorials/getting_started.md)
+    - [Feature Learning](./docs/zh_CN/application/feature_learning.md)
+- MOdel Prediction（Updating）
```
Comment (weisy11) [inline README_en.md:73]: Inference Model Prediction

File: README_en.md [hunk 9]
```diff
@@ -21,467 +21,97 @@ PaddleClas is a toolset for image classification tasks prepared for the industry
...
-| SqueezeNet1_1 | 0.601 | 0.819 | 0.76032       | 1.877      | 0.690   | 1.230 | [Download link](https://paddle-imagenet-models-name.bj.bcebos.com/dygraph/SqueezeNet1_1_pretrained.pdparams) |
-| VGG11 | 0.693 | 0.891 | 3.90412 | 9.51147 | 15.090 | 132.850 | [Download link](https://paddle-imagenet-models-name.bj.bcebos.com/dygraph/VGG11_pretrained.pdparams) |
-| VGG13 | 0.700 | 0.894 | 4.64684 | 12.61558 | 22.480 | 133.030 | [Download link](https://paddle-imagenet-models-name.bj.bcebos.com/dygraph/VGG13_pretrained.pdparams) |
-| VGG16 | 0.720 | 0.907 | 5.61769 | 16.40064 | 30.810 | 138.340 | [Download link](https://paddle-imagenet-models-name.bj.bcebos.com/dygraph/VGG16_pretrained.pdparams) |
-| VGG19 | 0.726 | 0.909 | 6.65221 | 20.4334 | 39.130 | 143.650 | [Download link](https://paddle-imagenet-models-name.bj.bcebos.com/dygraph/VGG19_pretrained.pdparams) |
-| DarkNet53 | 0.780 | 0.941 | 4.10829 | 12.1714 | 18.580 | 41.600 | [Download link](https://paddle-imagenet-models-name.bj.bcebos.com/dygraph/DarkNet53_pretrained.pdparams) |
+The steps of image recognition can be divided into three steps:
```
Comment (weisy11) [inline README_en.md:96]: The steps of 去掉吧

File: README_en.md [hunk 10]
```diff
@@ -21,467 +21,97 @@ PaddleClas is a toolset for image classification tasks prepared for the industry
...
+- （1）Identify region proposal for image objects through a detection model；
+ - （2）Extract the image features for each region proposal;
+- （3）Match the extracted features with the images in the retrieval database and output results;
 
+For a new unknown category, there is no need to retrain the model, just fill in the retrieval database with images of that category, recreate the retrieval database, and the category can be recognised.
```
Comment (weisy11) [inline README_en.md:102]: just prepare images of new category, extract features and update retrieval database

File: README_en.md (hunk 3 above)
Comment (lilith-zy) [reply README_en.md:30]: Rich library of pre-trained models: Provide a total of 134 ImageNet pre-trained models in 29 series, among which 6 selected series of models support fast structural modification.
那就改成这个吧？

Comment (lilith-zy) [reply README_en.md:30]: Rich library of pre-trained models: Provide a total of 134 ImageNet pre-trained models in 29 series, among which 6 selected series of models support fast structural modification.
那就改成这个吧？

Comment (lilith-zy) [reply README_en.md:30]: Rich library of pre-trained models: Provide a total of 134 ImageNet pre-trained models in 29 series, among which 6 selected series of models support fast structural modification.
那就改成这个吧？

Comment (lilith-zy) [reply README_en.md:30]: Rich library of pre-trained models: Provide a total of 134 ImageNet pre-trained models in 29 series, among which 6 selected series of models support fast structural modification.
那就改成这个吧？

File: README_en.md [hunk 11]
```diff
@@ -21,467 +21,97 @@ PaddleClas is a toolset for image classification tasks prepared for the industry
...
-| VGG11 | 0.693 | 0.891 | 3.90412 | 9.51147 | 15.090 | 132.850 | [Download link](https://paddle-imagenet-models-name.bj.bcebos.com/dygraph/VGG11_pretrained.pdparams) |
-| VGG13 | 0.700 | 0.894 | 4.64684 | 12.61558 | 22.480 | 133.030 | [Download link](https://paddle-imagenet-models-name.bj.bcebos.com/dygraph/VGG13_pretrained.pdparams) |
-| VGG16 | 0.720 | 0.907 | 5.61769 | 16.40064 | 30.810 | 138.340 | [Download link](https://paddle-imagenet-models-name.bj.bcebos.com/dygraph/VGG16_pretrained.pdparams) |
-| VGG19 | 0.726 | 0.909 | 6.65221 | 20.4334 | 39.130 | 143.650 | [Download link](https://paddle-imagenet-models-name.bj.bcebos.com/dygraph/VGG19_pretrained.pdparams) |
-| DarkNet53 | 0.780 | 0.941 | 4.10829 | 12.1714 | 18.580 | 41.600 | [Download link](https://paddle-imagenet-models-name.bj.bcebos.com/dygraph/DarkNet53_pretrained.pdparams) |
+The steps of image recognition can be divided into three steps:
+- （1）Identify region proposal for image objects through a detection model；
+ - （2）Extract the image features for each region proposal;
```
Comment (weisy11) [inline README_en.md:98]: extract features

File: README_en.md [hunk 12]
```diff
@@ -21,467 +21,97 @@ PaddleClas is a toolset for image classification tasks prepared for the industry
...
-| SqueezeNet1_1 | 0.601 | 0.819 | 0.76032       | 1.877      | 0.690   | 1.230 | [Download link](https://paddle-imagenet-models-name.bj.bcebos.com/dygraph/SqueezeNet1_1_pretrained.pdparams) |
-| VGG11 | 0.693 | 0.891 | 3.90412 | 9.51147 | 15.090 | 132.850 | [Download link](https://paddle-imagenet-models-name.bj.bcebos.com/dygraph/VGG11_pretrained.pdparams) |
-| VGG13 | 0.700 | 0.894 | 4.64684 | 12.61558 | 22.480 | 133.030 | [Download link](https://paddle-imagenet-models-name.bj.bcebos.com/dygraph/VGG13_pretrained.pdparams) |
-| VGG16 | 0.720 | 0.907 | 5.61769 | 16.40064 | 30.810 | 138.340 | [Download link](https://paddle-imagenet-models-name.bj.bcebos.com/dygraph/VGG16_pretrained.pdparams) |
-| VGG19 | 0.726 | 0.909 | 6.65221 | 20.4334 | 39.130 | 143.650 | [Download link](https://paddle-imagenet-models-name.bj.bcebos.com/dygraph/VGG19_pretrained.pdparams) |
-| DarkNet53 | 0.780 | 0.941 | 4.10829 | 12.1714 | 18.580 | 41.600 | [Download link](https://paddle-imagenet-models-name.bj.bcebos.com/dygraph/DarkNet53_pretrained.pdparams) |
+The steps of image recognition can be divided into three steps:
+- （1）Identify region proposal for image objects through a detection model；
```
Comment (weisy11) [inline README_en.md:97]: image objects -> target objects (虽然还是感觉不太顺)

File: README_en.md (hunk 1 above)
Comment (lilith-zy) [reply README_en.md:7]: 好的

File: README_en.md [hunk 13]
```diff
@@ -21,467 +21,97 @@ PaddleClas is a toolset for image classification tasks prepared for the industry
...
+ - （2）Extract the image features for each region proposal;
+- （3）Match the extracted features with the images in the retrieval database and output results;
```
Comment (weisy11) [inline README_en.md:99]: search features in the retrieval database and output results;

**Pull request closed without merging**